### PR TITLE
101 filebrowse tweaks

### PR DESCRIPTION
**Pull request by Aligorith**
This implements the following changes (as per #101):
* Make it possible to pass a folder-path as a command-line arg, instead of requiring a file within that folder (to avoid having a loading message that won't go away)
* Ctrl-Scrollwheel can be used to go to next/prev images
* Next/Prev images order is sorted using Natural / Lexical Sort Order
* Next/Prev images tools cycle around after hitting the first/last image in the folder, instead of getting stuck
* Ctrl-O / F1 opens a native file dialog to allow the user to select an image to load

I've also taken the liberty to add these to a new section at the top of the changelog. Not sure if you usually auto-generate / update this or not. It was just helpful to keep track of what I'd already implemented.

### CODE REVIEW

**Comment by woelper** [comment]
Thanks!
Yes, the changelog is auto-generated from conventional commits.

**Comment by woelper** [comment]
This is a really cool addition, and I think we should merge that as it is. I am thinking later if would be nice to have some more timeline-like widget for stepping through folders or image sequences. I imagine something like reading all image paths into a Vec so you can scrub through it or jump to specific frame, what do you think? This would allow us to do more sanitizing/filtering in a cleaner way.

**Comment by Aligorith** [comment]
Updated to remove the manual changelog changes that were causing merge conflicts.

File: src/main.rs [hunk 1]
```diff
@@ -906,3 +950,29 @@ fn next_image(state: &mut OculanteState) {
...
 }
+
+// Show file browser to select image to load
+fn browse_for_image_path(state: &mut OculanteState) {
+    let start_directory =
+        if let Some(img_path) = &state.current_path {
+            img_path.clone()
+        }
+        else {
+            std::env::current_dir().unwrap()
```
Comment (woelper) [inline src/main.rs:961]: This needs to be handled

**Review by woelper — APPROVED** [review]

**Pull request merged**